x-pack/solutions/security/plugins/security_solution/public/detections/hooks/alert_summary/use_fetch_integrations.ts

```diff
@@ -48,26 +43,17 @@ export interface UseFetchIntegrationsResult {
  * - crowdstrike
  */
 export const useFetchIntegrations = (): UseFetchIntegrationsResult => {
-  const { fleet } = useKibana().services;
-  const isAuthorizedToFetchSettings = fleet?.authz.fleet.readSettings;
-  const { isFetchedAfterMount: isSettingsFetched } = useGetSettingsQuery({
-    enabled: isAuthorizedToFetchSettings,
+  // TODO this might need to be revisited once the integration make it out of prerelease
+  //  The issue could be that we see both the prerelease and the release versions
```

Comment:
> The issue could be that we see both the prerelease and the release versions

The issue will be user will see prerelease versions and not the GA ones

Reply:
@nchaulet  Ah I see, thank you for pointing this out, I will correct the comment and bring this up to the team next sync.
What could be an option for me here? What happens if I want to show all the AI for SOC integrations, but some of them are prerelease and some are GA?

Reply:
> What could be an option for me here? What happens if I want to show all the AI for SOC integrations, but some of them are prerelease and some are GA?

I think we do not have an option to mix GA and prerelease packages, if we use the prerelease flag there is the risk that for an integration A in GA today, if someone publish a beta version of that package, your page will present the beta version of that integration, if it's an acceptable behaviour not sure there is so much issues.

If it's really a concern, we could imagine doing two queries, one for prerelease package, one for non prerelease, and merge the results, but maybe I am overthinking this.

Reply:
Ok thanks for the explanation. I will bring this to the rest of the team. At least we have an workaround and do like you suggested by fetching things twice.

Thank you!